unison-src/tests/methodical/dots.u

```diff
@@ -1,11 +1,11 @@
 
 -- You can define an operator called dot
-(.) f g x = f (g x)
+(`.`) f g x = f (g x)
```

Comment:
This is interesting; it makes me think that there are some unambiguous syntaxes we could accept without ``` `` ```, e.g. if it's in parens, all symbols, with no spaces, like the `(.)` above, or a group of symboly chars that are separated with spaces or other tokens, from any other identifiers, thus not a path-qualified name, like the `id . id` below, but this is still fine/good for now.